### ISSUE TITLE
[Documentation]: improve docs around IdP integration

### ISSUE THREAD

**Issue by sabineschaller** [interledger/rafiki#2641]
Right now, we only have a short paragraph about the IdP in the [Getting Started](https://rafiki.dev/integration/getting-started/#identity-provider) docs. They also reference some docs in the auth package, but there is no link. 

Additionally, we explain the interaction flow [here](https://rafiki.dev/concepts/open-payments/grant-interaction/). However, we don't link to it from the Getting Started page.

Finally, it is not clear from the Interaction Flow docs what the `id` is. Since the route is `/grant/:id/:nonce`, one may think it's the `grantId`. However, it is the `interactId`. This needs to be clarified. 

Long story short, we need to completely overhaul our docs on the IdP and how Rafiki interacts with it.
